packages/native/types/index.d.ts

```diff
@@ -0,0 +1,50 @@
+// Definitions by: Pat Sissons <https://github.com/patsissons>
+// TypeScript Version: 3.4
+
+import { Theme } from '@emotion/core'
+
+import {
+  CreateStyled as BaseCreateStyled,
+  CreateStyledComponent,
+  Interpolation,
+  ReactNativeStyle,
+  ReactNativeComponentNames,
+  ReactNativeComponentProps,
+  ReactNativeComponents,
+  ReactNativeStyleType
+} from './base'
+
+export {
+  ArrayInterpolation,
+  CreateStyledComponent,
+  FunctionInterpolation,
+  Interpolation,
+  ObjectInterpolation,
+  ReactNativeStyle,
+  StyledComponent,
+  StyledOptions
+} from './base'
+
+export function css<StyleType extends ReactNativeStyle = ReactNativeStyle>(
+  template: TemplateStringsArray,
+  ...args: Array<Interpolation>
```

Comment:
I'm not sure if this should use `Interpolation`. For instance, it doesn't make sense for `css` to accept functions - for the web we've split this into two similar, but separate, Interpolation and CSSInterpolation. I feel like the same should be done here.

Reply:
makes sense to me, i can add those suggestions in pretty quickly